fastlane/Fastfile

```diff
@@ -5,6 +5,34 @@ lane :test do
   validate_repo(tool_name: detect_tool)
 end
 
+desc "Test pull request"
```

Comment:
This `Fastfile` is just for `fastlane` deployment - please remove all the code you added here :+1: